packages/dds/tree/src/feature-libraries/sequence-field/config.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export enum CellOrderingMethod {
```

Comment:
In general, is there any documentation on these two schemes? If not, don't delay this work but I'd love an item filed to make this approachable to future devs.

Reply:
I'd love to make it approachable to current devs as well! I filed task 6667.